### ISSUE TITLE
ignore attributes on an c# Enum

### ISSUE THREAD

**Issue by chrizy** [Evertras/typescript-cs-poco#16]
Is there any way it could ignore attributes on an Enum,

```
   public enum SettingValueID
    {
        [SettingMetadata("2048")]
        MaxFileUploadSizeBytes = 1,

        [SettingMetadata(@"£")]
        Currency = 2,
  }
```
currently converts to

`declare enum SettingValueID { [SettingMetadata("2048")] = 0, MaxFileUploadSizeBytes = 1, [SettingMetadata(@"�")] = 2, Currency = 2 }`

Thanks

**Comment by Evertras** [comment]
Fixed in 1.3.8, should ignore attributes now.

**Comment by chrizy** [comment]
Thanks..!
